### PR TITLE
Add task for adding/moving forms in groups

### PR DESCRIPTION
**Pull request by lfdebrux**
### What problem does this pull request solve?

Trello card: https://trello.com/c/7EkOqWib/1605-add-rake-task-to-move-forms-from-one-group-to-another 

We expect that users will want to move forms from one group to another. As we don't yet have a design for users to do this themselves, this commit adds a Rake task that a developer can run as a support task.

### Things to consider when reviewing

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

**Review by thomasiles — DISMISSED** [review]
looks good!

I was worried that this is quite a sharp tool. Any form can be moved to any group regardless of form status/group status or form organisation/group organisation/creator organisation.

I don't think those worries are justified though. If we cause ourselves issues, we can change it.

**Comment by lfdebrux** [comment]
> Any form can be moved to any group regardless of form status/group status or form organisation/group organisation/creator organisation.

Would it be useful at this point to have it print more information about the new group/old group?

**Review by DavidBiddle — APPROVED** [review]
Code looks good and have tested successfully on my machine :tada:

File: lib/tasks/forms.rake [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  desc "move one or more forms into group"
+  task :move_dry_run, [] => :environment do |_, args|
+    *form_ids, group_id = args.to_a
+
+    usage_message = "usage: rake forms:move_dry_run[<form_id>, ..., <group_id>]".freeze
+    abort usage_message if form_ids.blank? || group_id.blank?
+
+    ActiveRecord::Base.transaction do
+      move_forms(form_ids, group_id)
+      raise ActiveRecord::Rollback
```
[the remark this answers is not in the document]
Comment (lfdebrux) [reply lib/tasks/forms.rake:23]: Oh, it would be good to log that there was a rollback though, I had thought of that but forgot to add it.

**Review by DavidBiddle — DISMISSED** [review]

**Comment by thomasiles** [comment]
> > Any form can be moved to any group regardless of form status/group status or form organisation/group organisation/creator organisation.
> 
> Would it be useful at this point to have it print more information about the new group/old group?

Yes, i think a quick summary of group (name, org, creator) might be useful - not vital to merging this though 👍

**Comment by lfdebrux** [comment]
> > > Any form can be moved to any group regardless of form status/group status or form organisation/group organisation/creator organisation.
> > 
> > 
> > Would it be useful at this point to have it print more information about the new group/old group?
> 
> Yes, i think a quick summary of group (name, org, creator) might be useful - not vital to merging this though 👍

I've added more logging in [8d36a14](https://github.com/alphagov/forms-admin/pull/1257/commits/8d36a14242d86a26ceda6102b503f810475ee6b7), let me know what you think.

**Review by thomasiles — APPROVED** [review]
awesome!

**Pull request merged**